apps/cowswap-frontend/src/legacy/state/lists/hooks.ts

```diff
@@ -104,20 +99,7 @@ export function useCombinedActiveList(): TokenAddressMap {
 
 // list of tokens not supported on interface for various reasons, used to show warnings and prevent swaps and adds
 export function useUnsupportedTokenList(): TokenAddressMap {
-  // get hard-coded broken tokens
-  const brokenListMap = useMemo(() => tokensToChainTokenMap(BROKEN_LIST), [])
```

Comment:
Actually, BROKEN_LIST and UNSUPPORTED_TOKEN_LIST are empty and we don't need them